nbnet.h

```diff
@@ -1728,15 +1728,12 @@ struct NBN_Driver
     NBN_DriverImplementation impl;
 };
 
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wmissing-field-initializers"
 static NBN_Driver nbn_drivers[NBN_MAX_DRIVERS] = {
-    {-1, NULL},
-    {-1, NULL},
-    {-1, NULL},
-    {-1, NULL}
+    { .id = -1 },
```

Comment:
maybe the structures should be fully initialized? I don't think the fields are necessarily going to be zero-initialized in this case.